src/reducers/maps.reducer.js

```diff
@@ -42,6 +43,14 @@ export default function maps(state = initialState.timeline, action) {
         ...state,
         selectedMapId: selectedMap ? selectedMap.id : state.selectedMapId
       }
+    case POI_SELECTED_FOR_PREVIEW:
```

Comment:
when is this needed? i don't think there should be a use case for being able to preview a poi for a map you're not on